rcldotnet/Exceptions.cs

```diff
@@ -0,0 +1,32 @@
+/* Copyright 2019 Francisco Martín <fmrico@gmail.com>
```

Comment:
Do you mind keeping the same copyright notice as the rest of the files? Thanks.

Reply:
Oooops. I was not sure of this rule. Now it is clear for me. I am very sorry about it.